public/css/about.css

```diff
@@ -212,6 +218,50 @@
   padding-top: 10px;
 }
 
+@media (max-width: 990px) {
+  .involvement {
+    text-align: center;
+  }
+}
+
+/*------------ SPAX -----------------*/
+
+.spax-pic {
+  width: 100%;
+  max-width: 500px;
+  padding-bottom: 25px;
+}
+
+.spax h1 {
+  border-bottom: solid black 10px;
+  display: inline-block;
+}
+
+.spax-about {
+  padding-top: 10px;
+  margin: 0;
+}
+
+.spax {
+  padding-top: 70px;
+  padding-bottom: 40px;
+  padding-right: 70px;
+  padding-left: 70px;
+  background-color: rgba(117, 120, 123, 0.2);
+}
+
+@media (max-width: 990px) {
+  .spax {
+    text-align: center;
+  }
+}
+
+@media (max-width: 990px) {
```

Comment:
This media query can be merged with the one above it since they're affecting the same element at the same screen size:

```
@media (max-width: 990px) {
  .spax {
     text-align: center;
     padding: 70px 30px;
  }
}
```